Tasks/PackerBuildV0/Tests/L0.ts

```diff
@@ -46,7 +46,17 @@ describe('PackerBuild Suite', function() {
             assert(tr.invokedToolCount == 4, 'should have invoked tool four times. actual: ' + tr.invokedToolCount);
             assert(tr.stdout.indexOf(match1) > -1, 'correctly writes contents of var file (containing azure spn details)');
             assert(tr.stdout.indexOf(match2) > -1, 'correctly writes contents of var file (containing template variables)');
+            done();
+        });
+
+        it('Runs successfully for windows template with managed image', (done:MochaDone) => {
+            let tp = path.join(__dirname, 'L0WindowsManaged.js');
+            let tr : ttm.MockTestRunner = new ttm.MockTestRunner(tp);
+            tr.run();
 
+            assert(tr.invokedToolCount == 4, 'should have invoked tool four times. actual: ' + tr.invokedToolCount);
+            assert(tr.stderr.length == 0 || tr.errorIssues.length, 'should not have written to stderr');
+            assert(tr.succeeded, 'task should have succeeded');        
```

Comment:
Add one more validation to ensure that image print in printed in output log

Reply:
added validation